app/views/comments/new.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="comment-container">
+<h3>Add a new Comment</h3>
+<div class="comment-form">
+<%= form_with model: [@user, @post, @comment], url: user_post_comments_path do |f| %>
+<div>
+<%= f.label :text, required: true%>
+<%= f.text_field :text, required: true %>
+</div>
+<%= f.submit "Submit" %>
+</div>
+</div>
+<% end %>
```

Comment:
- [ ] There is an example.com.png file that you are using in your project which is not in the image pipeline/folder in the assets, when I added it, there was progress and as a result, I will encourage you to also add it.
![image](https://github.com/firstpj/blog-app/assets/86200356/ab3541a3-789e-4be4-946d-75951f21f3e8)